CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.83.5-dev
```

Comment:
For `-dev` releases, don't we generally just put it in the pubspecs and put the regular release version in the changelogs?

Reply:
Now that we have automated tooling to bump them for us with no extra effort, I think it's a little nicer to keep everything consistent.